File: mobile/src/main/javaBackup/com/planit/home/planit007/CustomWindow.java
package planit007.planit.home.planit007;

import android.app.Activity;
import android.os.Bundle;
import android.view.Window;
import android.widget.TextView;

import com.planit.home.planit003.R;

/**
 * Created by Home on 2018-07-26.
 */

public class CustomWindow extends Activity {
    protected TextView title;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        requestWindowFeature(Window.FEATURE_CUSTOM_TITLE);

        setContentView(planit.planit.home.planit003.R.layout.activity_main);

        getWindow().setFeatureInt(Window.FEATURE_CUSTOM_TITLE, planit.planit.home.planit003.R.layout.window_title);

        title = (TextView) findViewById(planit.planit.home.planit003.R.id.title);
        title.setText("PlanIt");
    }
}

File: mobile/src/main/kotlin/com/planit/mobile/CustomWindow.kt
package com.planit.mobile

import android.app.Activity
import android.os.Bundle
import android.view.View
import android.view.Window
import android.widget.TextView

import com.planit.mobile.R

/**
 * Created by Home on 2018-07-26.
 */

class CustomWindow : Activity() {

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        var title: TextView

        requestWindowFeature(Window.FEATURE_CUSTOM_TITLE)

        setContentView(R.layout.activity_main)

        window.setFeatureInt(Window.FEATURE_CUSTOM_TITLE, R.layout.window_title)

        title = findViewById<TextView>(R.id.title)
        title.text = "PlanIt"

    }
}
